backend/src/main/kotlin/com/climatechangemakers/act/feature/findlegislator/manager/LegislatorsManager.kt

```diff
@@ -14,39 +15,43 @@ import kotlinx.coroutines.coroutineScope
 import javax.inject.Inject
 
 class LegislatorsManager @Inject constructor(
-  private val civicService: GoogleCivicInformationService,
+  private val geocodioService: GeocodioService,
   private val lcvScoreManager: LcvScoreManager,
 ) {
 
   suspend fun getLegislators(request: GetLegislatorsRequest): List<Legislator> = coroutineScope {
```

Comment:
Would be nice to make this function and its associated request more specific.  We might want to add other ways to look up legislators later.
```suggestion
  suspend fun getLegislatorsByAddress(request: GetLegislatorsByAddressRequest): List<Legislator> = coroutineScope {
```

Reply:
Do you have an example of what you're talking about? 

Generally, I don't try to over-engineer for a future which might not manifest. We could always provide an overloaded `getLegislators` function which takes, for example, a congressional district.

Reply:
Sorry didn't see your example. I like that.